### ISSUE TITLE
TracingEnabled not defined for resource of type AWS::Serverless::APi

### ISSUE THREAD

**Issue by vfrank66** [aws-cloudformation/cfn-python-lint#978]
*cfn-lint version: (`cfn-lint --version`)*

$ cfn-lint --version
cfn-lint 0.10.2

*Description of issue.*

Cfn lint is not recognizing the [TracingEnabled property](https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessapi) on the resource type AWS::Serverless::Api:

```
[cfn-lint] E0001:Resource with id [ExampleApi] is invalid. property TracingEnabled not defined for resource of type AWS::Serverless::Api
```

Please provide as much information as possible:
* Template linting issues:
  * Please provide a CloudFormation sample that generated the issue.
  * If present, please add links to the (official) documentation for clarification.
  * Validate if the issue still exists with the latest version of `cfn-lint` and/or the latest Spec files
* Feature request:
  * Please provide argumentation about the missing feature. Context is key!
```yaml
  ExampleApi:
    Type: AWS::Serverless::Api
    DependsOn: ExampleApiLamda
    Properties:
      Name: ExampleApi
      StageName: !Ref Environment
      DefinitionUri: example-swagger.yaml
      EndpointConfiguration: EDGE #PRIVATE
      TracingEnabled: true
      # AccessLogSetting:
      #   DestinationArn: !GetAtt ExampleLogsGroup.Arn
      Variables:
        LambdaFunctionName: !Ref ExampleApiLamda
```
Cfn-lint uses the [CloudFormation Resource Specifications](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) as the base to do validation. These files are included as part of the application version. Please update to the latest version of `cfn-lint` or update the spec files manually (`cfn-lint -u`)

__Unrelated but doing `cfn-lint-u` results in:__

<details>
$ cfn-lint -u 
2019-06-24 18:07:51,994 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region us-east-1
2019-06-24 18:07:52,727 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region us-west-1
2019-06-24 18:07:53,822 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region ap-northeast-3
2019-06-24 18:07:55,289 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region ap-northeast-2
2019-06-24 18:07:56,567 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region ap-northeast-1
2019-06-24 18:07:57,776 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region sa-east-1
2019-06-24 18:07:59,507 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region ap-southeast-1
2019-06-24 18:08:00,039 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region ca-central-1
2019-06-24 18:08:01,705 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region ap-southeast-2
2019-06-24 18:08:02,425 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region us-west-2
2019-06-24 18:08:02,850 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region us-east-2
2019-06-24 18:08:04,801 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region ap-south-1
2019-06-24 18:08:05,964 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region eu-central-1
2019-06-24 18:08:06,910 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region eu-west-1
2019-06-24 18:08:07,949 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region eu-west-2
2019-06-24 18:08:08,884 - cfnlint.maintenance - INFO - Patch not applied: VpcEndpointType in AWS::EC2::VPCEndpoint in region eu-west-3
</details>

**Comment by fatbasstard** [comment]
Looks like you're on a pretty old version (and probably also SAM).

Tried this with the latest version ([0.21.6](https://github.com/aws-cloudformation/cfn-python-lint/releases/tag/v0.21.6)) and it works there.

Can you update to the latest version of the linter?

**Comment by fatbasstard** [comment]
The `-u` option fails for the same reason, the "old" linter tries to apply patches are not valid with the latest spec files anymore

**Comment by vfrank66** [comment]
That fixed my issue sorry about that, forgot it was a pip package!
